### ISSUE TITLE
Behavior inside a Flatpak sandbox?

### ISSUE THREAD

**Issue by elsiehupp** [ActiveState/appdirs#168]
I don't know how to test this, and it isn't immediately clear to me from the code, but does this choose the correct directories within a Flatpak sandbox? According to [the Flatpak documentation](https://docs.flatpak.org/en/latest/conventions.html#xdg-base-directories), the environmental variables within a Flatpak sandbox should return as follows:

Base directory | Usage | Default location
-- | -- | --
`XDG_CONFIG_HOME` | User-specific configuration files | `~/.var/app/<app-id>/config`
`XDG_DATA_HOME` | User-specific data | `~/.var/app/<app-id>/data`
`XDG_CACHE_HOME` | Non-essential user-specific data | `~/.var/app/<app-id>/cache`

I don't know about `XDG` variables beyond these within the Flatpak sandbox.

Basically, it would be nice for the documentation (e.g. `README.md`) to explicitly state the behavior of `appdirs` within a Flatpak sandbox. (Similarly, the documentation could explicitly state the behavior within a [Snap sandbox](https://snapcraft.io/docs/environment-variables) and within a [macOS container](https://developer.apple.com/library/archive/documentation/Security/Conceptual/AppSandboxDesignGuide/AppSandboxInDepth/AppSandboxInDepth.html#//apple_ref/doc/uid/TP40011183-CH3-SW15), as well.)

I'd be happy to help if you can point me in the right direction.

Thanks!
